ui/src/components/App.js

```diff
@@ -10,7 +10,6 @@ class App extends React.Component {
       <Router>
         <div className="App">
           <Switch>
-            import AppDetailedView from './AppDetailedView';
```

Comment:
this was accidentally committed and merged with previous PR, and just removing here